tests/conftest.py

```diff
@@ -46,7 +46,7 @@ def pytest_addoption(parser: pytest.Parser):
 LOCALHOST = "127.0.0.1"
 OPEN_CHANNEL_FUNDING_VALUE_HOPR = 1000
 
-TICKET_AGGREGATION_THRESHOLD = 100
```

Comment:
Unless the tests also send more messages, they will keep failing. 

Why not doing it the other way around:

Configure the test nodes to use 100 as agg threshold as they used to when it was the default? That won't require sending more messages in tests and they won't need to run longer.

Reply:
I agree, but the tests fails also with 100. And I did not manage to have the test running locally: the nodes never boot. I assumed that in the CI it could work, but apparently no. I will look again